### PR TITLE
Replace past.builtins.xrange for range

### PR DESCRIPTION
**Pull request by saullocarvalho**

### CODE REVIEW

File: claripy/ast/bool.py [hunk 1]
```diff
@@ -182,7 +181,7 @@ def constraint_to_si(expr):
     satisfiable, replace_list = backends.vsa.constraint_to_si(expr)
 
     # Make sure the replace_list are all ast.bvs
-    for i in xrange(len(replace_list)):
+    for i in range(len(replace_list)):
```
Comment (ltfish) [inline claripy/ast/bool.py]: Please add `# pylint:disable=consider-using-enumerate` at the end of this line to stop pylint from complaining.

Comment (ltfish) [reply claripy/ast/bool.py]: See https://travis-ci.org/angr/claripy/jobs/533078142.

Comment (saullocarvalho) [reply claripy/ast/bool.py]: Is not better just use `enumerate` as suggested?

Comment (saullocarvalho) [reply claripy/ast/bool.py]: Ok!

Comment (ltfish) [reply claripy/ast/bool.py]: See line 188. We are modifying the list in place.

Comment (saullocarvalho) [reply claripy/ast/bool.py]: I missed that. I am adding the comment.
Thanks!

**Comment by ltfish** [comment]
Thank you!

**Pull request merged**